spirv_cross_c.h

```diff
@@ -293,22 +293,24 @@ typedef enum spvc_msl_index_type
 /* Maps to C++ API. */
 typedef enum spvc_msl_shader_variable_format
 {
+
 	SPVC_MSL_SHADER_VARIABLE_FORMAT_OTHER = 0,
 	SPVC_MSL_SHADER_VARIABLE_FORMAT_UINT8 = 1,
 	SPVC_MSL_SHADER_VARIABLE_FORMAT_UINT16 = 2,
-	SPVC_MSL_SHADER_VARIABLE_FORMAT_ANY16 = 3,
-	SPVC_MSL_SHADER_VARIABLE_FORMAT_ANY32 = 4,
-
-	/* Deprecated names. */
+	SPVC_MSL_SHADER_VARIABLE_FORMAT_UINT32 = 3,
+	SPVC_MSL_SHADER_VARIABLE_FORMAT_FLOAT = 4,
+	SPVC_MSL_SHADER_VARIABLE_FORMAT_INT8 = 5,
+	SPVC_MSL_SHADER_VARIABLE_FORMAT_INT16 = 6,
+	SPVC_MSL_SHADER_VARIABLE_FORMAT_INT32 = 7,
+	SPVC_MSL_SHADER_VARIABLE_FORMAT_HALF = 8,
+
+	// Deprecated aliases.
 	SPVC_MSL_VERTEX_FORMAT_OTHER = SPVC_MSL_SHADER_VARIABLE_FORMAT_OTHER,
 	SPVC_MSL_VERTEX_FORMAT_UINT8 = SPVC_MSL_SHADER_VARIABLE_FORMAT_UINT8,
 	SPVC_MSL_VERTEX_FORMAT_UINT16 = SPVC_MSL_SHADER_VARIABLE_FORMAT_UINT16,
 	SPVC_MSL_SHADER_INPUT_FORMAT_OTHER = SPVC_MSL_SHADER_VARIABLE_FORMAT_OTHER,
 	SPVC_MSL_SHADER_INPUT_FORMAT_UINT8 = SPVC_MSL_SHADER_VARIABLE_FORMAT_UINT8,
 	SPVC_MSL_SHADER_INPUT_FORMAT_UINT16 = SPVC_MSL_SHADER_VARIABLE_FORMAT_UINT16,
-	SPVC_MSL_SHADER_INPUT_FORMAT_ANY16 = SPVC_MSL_SHADER_VARIABLE_FORMAT_ANY16,
```

Comment:
This is a breaking change, isn't it?

Reply:
I restored the any's, if anything, to keep existing tests from breaking.